src/rest_api/classes/laboratory/widgets/gene_classes.clj

```diff
@@ -0,0 +1,59 @@
+(ns rest-api.classes.laboratory.widgets.gene-classes
+  (:require
+    [datomic.api :as d]
+    [pseudoace.utils :as pace-utils]
+    [rest-api.classes.generic-fields :as generic]))
+
+(defn former-gene-classes [laboratory]
+  (let [db (d/entity-db laboratory)
+        data (->> (d/q '[:find ?gene-class ?description
+                         :in $ ?laboratory
+                         :where 
+                         [?laboratoryent :gene-class.former-designating-laboratory/laboratory ?laboratory]
+                         [?gc :gene-class/former-designating-laboratory ?laboratoryent]
+                         [?gc :gene-class/id ?gene-class]
+                         [?gc :gene-class/description ?description]]
+                       db (:db/id laboratory))
+                  (map (fn [oid]
+                         (let [gene-class (get oid 0)
+                               description (get oid 1)]
+                           (pace-utils/vmap
```

Comment:
Not clear why `pace-uitils/vmap` is being used here?
Could simply use a plain map? The only case for using `vmap` here would be if you only create key/value pairs when the value is not nil (but the other keys with hard-coded values would still exist so...)
```clojure
{:description description
 :former_gene_class
 {:taxonomy "all"
  :class "gene_class"
  :label "gene-class"
  :id gene-class}}
```

Reply:
Good to know, didn't realize the difference between vmap and just a map.  Fixed.